src/server/api/controllers/waypoints.controller.js

```diff
@@ -0,0 +1,81 @@
+const HttpError = require("../lib/utils/http-error");
+const knex = require("../../config/db");
+
+// Method for getting waypoints by id:
+const getWaypointsById = async id => {
+  try {
+    const waypoints = await knex("waypoints")
+      .select("id", "lon", "lat", "created_at", "updated_at")
```

Comment:
Sorry, but I merged Omids PR renaming to "longitude" and "latitude" respectively first, so you need to pull in the latest develop and rename these.